02-oop-inheritance/js/DownloadableMovie.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
 
-var DownloadableMovie = function () {
+var DownloadableMovie = (function () {
 
-  Movie.call(this);
+  /**
+   * Constructor  
+   */ 
+  function DownloadableMovie() {
+    Movie.call(this);
+  }
+
+  DownloadableMovie.prototype = new Movie();
```

Comment:
realmente es necesario redefinir el protorype de esta clase??
al llamar a Movie.call(this); supuestamente le estoy dando el closure de movie a DownloadableMovie y esta pasaria a tener todos los metodos de Movie, me suena a un poco redundante, o ahi algo q no estoy viendo :(